### PR TITLE
[ui] - reproducer for #gh-4182

### PR DESCRIPTION
**Pull request by mkralik3**
//test: `@integrations-webhook-db`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/integrations/webhook-db.feature [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+@database
+@datamapper
+@integrations-webhook-db
+Feature: Integration - Webhook to DB
+
+  Background: Clean application state
+    Given clean application state
+    Given log into the Syndesis
+
+    Given reset content of "contact" table
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/webhook-db.feature]: twice `And`

File: ui-tests/src/test/resources/features/integrations/webhook-db.feature [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    And set integration name "Webhook to DB with DB"
+    And publish integration
+
+    And navigate to the "Integrations" page
+    And wait until integration "Webhook to DB with DB" gets into "Running" state
+    And select the "Webhook to DB with DB" integration
+    And invoke post request to webhook with body {"first_name":"John","company":"Red Hat"}
+
+    Then checks that query "select * from contact where first_name='middleSTEP'" has 1 row output
+    And checks that query "select * from contact where first_name='John'" has 1 row output
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/webhook-db.feature]: add a new line here

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**